addons/web_editor/static/tests/field_html_tests.js

```diff
@@ -290,6 +290,7 @@ QUnit.module('web_editor', {}, function () {
             });
 
             await testUtils.form.clickEdit(form);
+            await new Promise(resolve => setTimeout(resolve, 50));
```

Comment:
Because otherwise this test fails on my computer (with or without the changes introduced by this PR).

![image](https://github.com/odoo/odoo/assets/65516503/c1fd466c-c115-4086-a428-06b7b1645653)

My guess is the web client is not yet ready after switching to edit mode, and looking for '.note-editable p' finds nothing.

![image](https://github.com/odoo/odoo/assets/65516503/3a4ab639-ecda-458e-89bc-8781779a1d79)

I suppose this is not a problem on a faster machine. It does not seem to be a problem on runbot.

By the way, in the same method, some 20 lines below, we have `await new Promise((resolve)=>setTimeout(resolve, 50))`.

Reply:
Would you like me to put `await new Promise(resolve => setTimeout(resolve, 50));` in a different commit?

Reply:
I did not know that this was exactly the same timeout line that already appeared below in the same test. Have you tried moving the one from line 315 up here rather than adding a new one ? Just wondering, as it might be possible that some changes were made that moved the asynchronicity from one point of the code to another. It would probably be the case in `master` with all the owl changes and such. It is less likely to be the case in 15.0 but I'd like you to try nonetheless, please. If it doesn't make the test pass then your new timeout can stay as is in the PR.

Reply:
Moving the timeout from line 315 does the trick, that is, the test passes without the need for a second timeout.